skema/program_analysis/CAST/python/ts2cast.py

```diff
@@ -80,6 +90,8 @@ def run(self, root) -> List[Module]:
         # Which can then contain multiple things (Handled at module visitor)
         return self.visit(root)
 
+    # TODO: node helper for ignoring comments
```

Comment:
If you haven’t already seen it, earlier this week I merged an update to the Fortran node_helper script that adds support for removing comments

Reply:
Oh, that's helpful! I'll look into that. Thanks!